examples/windowmediaplayer.py

```diff
@@ -52,26 +51,24 @@ def windows_media():
     app = application.Application()
 
     try:
-        app.start(   # connect_(path =
-            r"C:\Program Files\Windows Media Player\wmplayer.exe")
+        app.start(r"C:\Program Files (x86)\Windows Media Player\wmplayer.exe")
     except application.ProcessNotFoundError:
-        print("You must first start Windows Media "\
-            "Player before running this script")
+        print("You must first start Windows Media Player before running this script")
         sys.exit()
 
-    app.WindowsMediaPlayer.menu_select("View->GoTo->Library")
+    # Wait for Window media player displays
+    time.sleep(3)

```

Comment:
Such a hard-coded `time.sleep()` calls make the script a bit slow. It's better to call `app.WindowsMediaPlayer.SomeControlAppearedAfterPreviousAction.wait('ready')` (default timeout is up to 5 seconds). I will think if it's possible to do something inside `menu_select`.